src/app/profiles/device-profiles/device-profiles-list/device-profiles-list.component.ts

```diff
@@ -25,6 +24,7 @@ export class DeviceProfilesListComponent implements OnInit, OnDestroy {
   public errorMessages: any;
   deleteDialogSubscription: Subscription;
   errorTitle: string;
+  isDeleting = true;
```

Comment:
Don't default true

Reply:
I will remove this isDeleting stuff again. Took inspiration from other places in the code, but not sure it's needed anyways. And not sure it was actually working.